src/mono/wasi/Wasi.Build.Tests/WasiTemplateTests.cs

```diff
@@ -21,24 +21,126 @@ public WasiTemplateTests(ITestOutputHelper output, SharedBuildPerTestClassFixtur
     }
 
     [Theory]
-    [InlineData("Debug", /*aot*/ false)]
-    [InlineData("Debug", /*aot*/ true)]
-    [InlineData("Release", /*aot*/ false)]
-    [InlineData("Release", /*aot*/ true)]
-    public void ConsoleBuildThenPublish(string config, bool aot)
+    [InlineData("Debug", /*aot*/ false, /*singleFileBundle*/ false)]
+    [InlineData("Debug", /*aot*/ true, /*singleFileBundle*/ false)]
+    [InlineData("Release", /*aot*/ false, /*singleFileBundle*/ false)]
+    [InlineData("Release", /*aot*/ true, /*singleFileBundle*/ false)]
+    [InlineData("Debug", /*aot*/ false, /*singleFileBundle*/ true)]
+    // [InlineData("Debug", /*aot*/ true, /*singleFileBundle*/ true)] // https://github.com/dotnet/runtime/issues/95273
+    [InlineData("Release", /*aot*/ false, /*singleFileBundle*/ true)]
+    // [InlineData("Release", /*aot*/ true, /*singleFileBundle*/ true)] // https://github.com/dotnet/runtime/issues/95273
+    public void ConsoleBuild(string config, bool aot, bool singleFileBundle)
```

Comment:
This seems redundant since `ConsoleBuildThenPublish` is already being added below.

Reply:
> Another test to add - project file has RunAOTcompilation=true, but the test only runs build, and not publish.

That's this test. Only build, no publish. Now I'm confused

Reply:
Then you can remove the aot parameter